.gitignore

```diff
@@ -0,0 +1,3 @@
+.terraform
```

Comment:
Do you have a global gitignore file?

Reply:
Yes I have it, but that file is not included. 😄 

I added it here because while working with code under `examples` to test the feature this folder was created and I thought that it may be handy for other people working on the repo to have it ignored.

My rule of thumb for this kind of things is: if the file is being created by something *inside* the repository than it's worthy to be included in the repo's `.gitignore`.

If we don't agree on this, I can remove it. 😄